docs/tutorial/index.md

```diff
@@ -1,22 +1,30 @@
 ---
-title: Gatsby.js Tutorial
+title: Gatsby.js Tutorials
 ---
 
 Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site.
 
 > The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/quick-start/) page.
 
+## Gatsby fundamentals
+
 0.  [Set Up Your Development Environment](/tutorial/part-zero/): We'll introduce you to core technologies that power Gatsby, and guide you through setting up your development environment.
 1.  [Get to know Gatsby building blocks](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
 1.  [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
 1.  [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
 
-## Advanced tutorials
+## Intermediate tutorials
```

Comment:
oh nice, I like these changes. Fwiw, I don't think the 1-3 parts of tutorial are beginner. I agree that they are just fundamental. Part 0 could be considered beginner and moved somewhere else if it is too beginner for the main tutorial. Not sure how many people look at it! I had a huge part in writing it and yet never measured how successful it was...

The only question here is will the /tutorial/ sidebar lead people thru the new "advanced tutorials" as if they are part of the main tutorial? That might work ok, although the flow could be a little strange since they change in tone. Also the bottom of each doc in the main tutorial has a link to the next part of the tutorial, so that consistency would be lost and need work to update

Reply:
Hey @shannonbux, great points! Here are some screenshots to show how it's set up right now. The Advanced Tutorials sidebar item is still linked from Docs, and on the Tutorials page it's separate from steps 0-8. The Advanced Tutorials are listed on their own page and linked from the Tutorial introduction. I'll take a look at the copy at the end of the main part of the tutorial, and we can make additional improvements after this too (like adding more of a flow–I think this is sort-of a first pass on advanced tutorials, especially with the major changes planned for "Choose your own adventure" tutorials).

<img width="324" alt="Tutorials sidebar" src="https://user-images.githubusercontent.com/1045233/57944794-f4d70e80-788c-11e9-884f-19a23471e8ea.png">

<img width="1642" alt="Tutorials page" src="https://user-images.githubusercontent.com/1045233/57944806-002a3a00-788d-11e9-9f43-cd6a4ec7a9f9.png">

Reply:
This looks great! Do we need an "INTERMEDIATE TUTORIALS" sidebar grouping on the tutorials page to complement the "ADVANCED TUTORIALS" section?

Reply:
Beginner and intermediate are more of a group, as they go in order from 0-8. Advanced Tutorials has a separate heading in the sidebar because those don't yet follow the same flow...and people wouldn't necessarily do them in a specific order. They are more a la carte!

Given we have a future plan to redesign the tutorials to a "Choose Your Own Adventure" based on skill and experience level as starting points, I would hesitate to put too much effort into changing the existing structure. The biggest thing holding it back is the fact that sidebar grouping would collapse the tutorial items. Is there a way to add styled headers to the sidebar without adding "Gatsby fundamentals" and "Intermediate tutorials" pages? That would be overkill IMO.